hypothesis-python/src/hypothesis/core.py

```diff
@@ -76,9 +76,13 @@
     from coverage.collector import FileDisposition
 
 if False:
-    from typing import Any, Dict, Callable, Optional, Union  # noqa
+    from typing import (  # noqa
+        Any, Dict, Callable, Hashable, Optional, Union, TypeVar,  # noqa
```

Comment:
One `noqa` is not enough? IIRC, first one at import line should cover the rest import statement.

Reply:
Ah yes it does, thank you. I only tried it on the last line and assumed all lines needed it.